docs/content/en/install/doks/_index.md

```diff
@@ -0,0 +1,332 @@
+Server Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.7", GitCommit:"132a687512d7fb058d0f5890f07d4121b3f0a2e2", GitTreeState:"clean", BuildDate:"2021-05-12T12:32:49Z", GoVersion:"go1.15.12", Compiler:"gc", Platform:"linux/amd64"}
+```
+
+### Taint Nodes
+
+With mixed node types available we should apply taints to each node type in order to facilitate scheduling. The larger instances will be tainted with the value `app=cassandra:NoSchedule` effectively blocking the scheduling of Pods unless they have the taint `app=cassandra`. In our example deployment here Cassandra nodes are part of the pool named `pool-hluacyqq4`.
+
+```console
+# Taint Cassandra nodes
+kubectl taint node -l doks.digitalocean.com/node-pool=pool-hluacyqq4 app=cassandra:NoSchedule
+```
+
+**Output:**
+
+```console
+node/pool-hluacyqq4-8edt2 tainted
+node/pool-hluacyqq4-8edtl tainted
+node/pool-hluacyqq4-8edtt tainted
+```
+
+{{% alert title="Important" color="primary" %}}
+Your node pool names will most likely not match the values above. Make sure you adjust the commands / values appropriately for your environment.
+{{% /alert %}}
+
+## Install K8ssandra
+
+With all of the infrastructure provisioned we can now focus on installing K8ssandra. This will require configuring a service account for the backup and restore service and creating a set of Helm variable overrides.
+
+### Create Backup / Restore Service Account Secrets
+
+In order to allow for backup and restore operations, we must create a service account for the Medusa operator which handles coordinating the movement of data to and from DigitalOcean Spaces. As part of the provisioning section a key was generated for this purpose. Plug in the key and secret from the provisioning section in the following file.
+
+{{< readfilerel file="spaces.secret.yaml" highlight="yaml" >}}
+
+Next submit this file to Kubernetes as a secret.
+```bash
+kubectl create secret generic prod-k8ssandra-medusa-key --from-file=medusa_s3_credentials=./medusa_s3_credentials
+```
+
+**Output**:
+
+```bash
+secret/prod-k8ssandra-medusa-key created
+```
+
+{{% alert title="Important" color="primary" %}}
+The name of the key file within the secret MUST be `medusa_s3_credentials`. _Any_ other value will result in Medusa not finding the secret and backups failing.
+{{% /alert %}}
+
+This secret, `prod-k8ssandra-medusa-key`, can now be referenced in our K8ssandra configuration to allow for backing up data to DigitalOcean Spaces with Medusa.
+
+### Create `WaitForFirstConsumer` Storage Class
+
+With K8ssandra it is recommended that a Kubernetes Storage Class with `volumeBindingMode: WaitForFirstConsumer`. By default the pre-installed `do-block-storage` storage class uses `Immediate`. We will create a new storage class with this parameter changed for use with K8ssandra. First retrieve the existing storage class:
```

Comment:
recommended, or required?
Also, the first sentence seems incomplete. Suggest:

K8ssandra requires a Kubernetes Storage Class that has a`volumeBindingMode: WaitForFirstConsumer`. The default pre-installed `do-block-storage` storage class has `volumeBindingMode: Immediate`. We will create a new storage class with the required mode. First retrieve the existing storage class: